htmlCssTasks/sass/First_Example/content.html

```diff
@@ -0,0 +1,11 @@
+<html>
+   <head>
+      <title> Import example of sass</title>
+      <link rel = "stylesheet" type = "text/css" href = "output.css"/>
```

Comment:
Make sure, not to give space between attribute and its value.
rel = "stylesheet"  should be  rel="stylesheet"